tests/integ/test_function.py

```diff
@@ -2302,3 +2303,23 @@ def test_snowflake_cortex_summarize(session):
     # this length check is to get around the fact that this function may not be deterministic
     assert 0 < len(summary_from_col) < len(content)
     assert 0 < len(summary_from_str) < len(content)
+
+
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="FEAT: snowflake_cortex functions not supported",
+)
+def test_apply_snowflake_cortex_sentiment(session):
+    # TODO: SNOW-1758914 snowflake.cortex.sentiment error on GCP
+    if session.connection.host == "sfctest0.us-central1.gcp.snowflakecomputing.com":
```

Comment:
we can use this in the tests/integ/modin/test_apply_snowpark_python_functions.py too

Reply:
Should I also update `test_snowflake_cortex_summarize` in the meantime?

Reply:
That will be helpful! You can either update in this PR or a new PR.